course_info/tests/jasmine/controllers/PeopleControllerSpec.js

```diff
@@ -0,0 +1,125 @@
+describe('Unit testing PeopleController', function() {
+    var $controller, $rootScope, $routeParams, courseInstances, $compile, djangoUrl,
+        $httpBackend, $window;
+    var controller, scope;
+
+    beforeEach(function() {
+        module('CourseInfo');
+        inject(function(_$controller_, _$rootScope_, _$routeParams_, _courseInstances_,
+                        _$compile_, _djangoUrl_, _$httpBackend_, _$window_) {
+            $controller = _$controller_;
+            $rootScope = _$rootScope_;
+            $routeParams = _$routeParams_;
+            courseInstances = _courseInstances_;
+            $compile = _$compile_;
+            djangoUrl = _djangoUrl_;
+            $httpBackend = _$httpBackend_;
+            $window = _$window_;
+
+            // this comes from django_auth_lti, just stub it out so that the $httpBackend
+            // sanity checks in afterEach() don't fail
+            $window.globals = {
```

Comment:
Yeah...it's worth investigating a base class for ajax tests the next time around.  And, hell...forgot about the `verify` calls...adding them in.